### DISCUSSION TITLE
Self-Introductions

### DISCUSSION THREAD

**Discussion by pedrohbraga** [SORTEE-Github-Hackathon/manuscript#4] [General]
*Name*: Pedro Henrique Pereira Braga

*What do I do*: I am a Ph.D. Candidate in Biology at Concordia University, and I study how the role of spatial and temporal scale-dependence in processes maintaining and forming biodiversity patterns. I have been using bats 🦇 to test most of my hypotheses!

*How can I or with what I would like to contribute*: I work a lot with biostatistics and I work very often with Github. I am happy with contributing with statistics, writing and with the organization and test of Github tools.

*What is the most trepidation that I have*: Writing this type of suggestion/call for action paper is fairly new to me. I am also finishing my Ph.D. and finding time to do well everything that I would like to is a major challenge.

**Comment by DrMattG** [comment]
*Name*:  Matt Grainger

*What do I do*: I am a researcher at the Norwegian Institute for Nature Research. I do anything and everything! My interest is in pragmatic evidence synthesis to help conservation managers make decisions. I work on several projects to build tools to help systematic reviewers carry out reviews and tools to help researchers share data and metadata in accordance with the FAIR principles (mostly R based tools). 

*How can I or with what I would like to contribute*: I work with GitHub quite a lot (but by no means an expert). I am happy writing and producing figures (basically anything where I can be useful!)

*What is the most trepidation that I have*: I suppose a lack of time (which we all probably feel) but I am looking forward to writing collaboratively in this format - It is exciting

**Reply by wavingtowaves** [reply]
Agreed with you Matt, I'm really excited to collaborate with this team on a manuscript using GitHub :)

**Comment by shafieisabets** [comment]
*Name*: Saeed Shafiei Sabet

*What do I do*: Currently I am a research fellow and reader in Animal Behaviour at University of Guilan, Iran. I received my PhD at Leiden University, the Netherlands. In the next step of my career, I applied for a research fellowship program in Germany. So I am a kind of migratory researcher/species :) My research keywords are Anthropogenic Sound, Animal Behaviour, Bioacoustics, Fish, Crustaceans, Predator-Prey interactions, Anti-Predatory Behaviour. 

*How can I or with what I would like to contribute*: I love collaborations and working together in multicultural disciplines.  I would be happy to help and contribute with writing the introduction and discussion parts also help to develop figures and tables related to our manuscript.

*What is the most trepidation that I have*: As a researcher, I miss doing research as usual with no restriction because of the pandemic! Also, keep continuously communicating with other scientists. I am really happy and enthusiastic about our collaboration :)

**Reply by wavingtowaves** [reply]
That's cool that you research animal behavior. I started off in a grad program for animal behavior in NYC, but switched midway through to invasion ecology. That being said, I still remember enjoying lots of the papers/books I read in those first few years of animal behavior research.

**Reply by shafieisabets** [reply]
Cool Rob! Good to know!  :) Thank you for your reply. Maybe it will open new opportunities in our group for further multidisciplinary scientific collaborations! I also like to explore anthropogenic sound effects on invasive species as well. We are currently doing research to assess how anthropogenic sound may affect feedifing performance of the ornamental red cheery shrimp, an invasive species, under laboratory conditions.

**Reply by dylangomes** [reply]
Saeed, we have so much overlap, as I think we've previously discussed! Of your research keywords, I think only "Crustaceans" do not fit for me.

**Reply by shafieisabets** [reply]
> Saeed, we have so much overlap, as I think we've previously discussed! Of your research keywords, I think only "Crustaceans" do not fit for me.

Hi Dylan, sorry for my late reply. I think I just overlooked your message! Good to hear from you and Yes indeed. I hope this collaborative project would be a new starting point for our other joint relevant scientific activities. In terms of research keywords, I am thinking to extend and adding marine mammals as well in future (to be explored in The Persian Gulf as there is no relevant behavioural documentation yet).  I really like the international research team atmosphere and collaboration in my work. :)
Kind regards
Saeed

**Comment by vjf2** [comment]
_Name:_ Vivienne Foroughirad

_What do I do:_ I am a Postdoctoral Fellow in Biology at Georgetown University, and I study cetacean🐬 behavioral ecology and genetics. Most of my work is with bottlenose dolphins at a long-term field site in Shark Bay, Western Australia. 

_How can I or with what I would like to contribute:_ I am happy to help on various parts of writing, but I’m definitely interested in working on resources for documentation and non-coding aspects of repository design, since that’s something I feel like I never got good training in myself.    

_What is the most trepidation that I have:_ Most of my work in GitHub has been a solo operation so far, but I’m excited to use it more collaboratively now!

**Reply by dylangomes** [reply]
Cetaceans are super cool!

> Most of my work in GitHub has been a solo operation so far

Totally! Mine too!

**Comment by Aguncan** [comment]
Name: Ali Guncan

What do I do: I am an Entomologist and currently working at the Ordu University as an Associate Professor. My current research topics are Insect Ecology and Pest Management and recently interested on Tools used for Open and Reproducible Science.

How can I or with what I would like to contribute: I'd like to contribute drafting the manuscript and helping figures through my skills and experiences.

What is the most trepidation that I have: Lack of time

**Reply by dylangomes** [reply]
While I've worked with a lot of different taxa, I am most inspired when working with insects (just so diverse and so little is known)!

**Reply by Aguncan** [reply]
Yes, so diverse taxa, so many unusual traits or behaviors, very fascinating creatures.

**Comment by wavingtowaves** [comment]
*My Name is*:  Rob Crystal-Ornelas

*What do I do*: I'm a postdoc at Lawrence Berkeley National Lab in Berkeley, California.  At Berkeley Lab, I work for an environmental data repository called ESS-DIVE which stores all types of earth/environmental science :seedling:  :earth_americas: data.  Recently, I've been working with teams of researchers to help them develop guidelines and templates to help their teams and colleagues more consistency format data.

*How can I or with what I would like to contribute*: I'm looking forward to working on several things. First, I really enjoy coordinating our monthly-ish meetings where we brainstorm, bring up any questions we have about the manuscript etc. and I look forward to continuing those. I'm also excited to work on continuing to clarify the message and focus of our paper in the introduction, and of course pick a couple of topics to write about in the main body of the paper.

*What is the most trepidation that I have*: GitHub can seem like an overwhelming topic for non-programmers to start interacting with, so I want to make sure the manuscript really helps to get ecologists more familiar with the platform.

**Reply by shafieisabets** [reply]
This would be great! Then I would love to download and read the manuscript that we are writing when it has been published asap. :)

**Comment by colebrookson** [comment]
*Name:* Cole Brookson

*What do I do:* I'm a graduate student at the University of Alberta, where I work mostly on food web theory, developing computational approaches to understanding how and why food webs may rewire with increasing anthropogenic change. I also work a bit on disease dynamics, and contribute to open source software on the side.  

*How can I or with what I would like to contribute:* I work with Github a lot to contribute to various projects, and I'm really excited to work on the "good enough" components of the manuscript and any accompanying tutorials/demos/repos etc. I know that often ecologists get overwhelmed with yet another technology they're told they need to know, so I'm especially looking forward to developing content to help folks get started.

*What is the most trepidation that I have:* I do feel a bit of imposter syndrome writing this style of recommendation paper as I know that while I do use Github constantly, I am not a software developer and I am always a bit worried of coming off as though I know more than I do.

**Reply by dylangomes** [reply]
I am guessing that we all, too, have experienced imposter syndrome. I know I do all the time - so you're not alone!

**Reply by kaitlyngaynor** [reply]
Definitely not alone :) I think there's some advantage to writing this from a non-expert perspective, and we can ask some GitHub users outside of ecology & evolution for a friendly review of this before we submit, too, to see if we've missed something huge!

**Comment by BrandonEdwards** [comment]
*My Name is*:  Brandon Edwards

*What do I do*: PhD Candidate at Carleton University/Environment and Climate Change Canada. My current research focuses on developing an open-access database of detectability estimates for as many North American landbirds as possible, while trying to improve on the detectability estimates as much as possible by using as much available data as I can.

*How can I or with what I would like to contribute*: I am very excited to help draft the manuscript where I can, and to bring my extensive experience with Github to the manuscript as we point the eco evo community toward the direction of open science with Github.

*What is the most trepidation that I have*: I have not collaborated on a manuscript using Git with this many authors before, so I am both excited and curious to see how it will turn out!

🦆

**Comment by Aariq** [comment]
_Name_: Eric R. Scott

_What do I do_: I am a postdoc at UF in the [Bruna Lab](https://github.com/BrunaLab) currently modeling plant demography under climate change.  I am remote, living in Boston, so I have been working on manuscripts through GitHub.  In the past, I've also collaborated on the [webchem](https://docs.ropensci.org/webchem/) R package and an [accompanying manuscript](https://github.com/EDiLD/webchem_paper) through GitHub.

_How can I or with what I would like to contribute:_ I'm interested in contributing programming (e.g. to streamline writing or CI) and writing.  In particular, I like the idea of a box or table describing different "personas" who might be involved with a GitHub-hosted project, along with examples of how they can interact with GitHub with minimal additional learning.

_What is the most trepidation that I have:_ Limited time, and working on a project with so many collaborators (although, this will be a good example for our paper!)

**Comment by kaitlyngaynor** [comment]
*My Name is*: Kaitlyn Gaynor

*What do I do*: I am a postdoc at the National Center for Ecological Analysis and Synthesis ([NCEAS](https://www.nceas.ucsb.edu)) in Santa Barbara, California, and soon-to-be assistant professor at the University of British Columbia in Vancouver. I [work on a range of topics](http://www.kaitlyngaynor.com/research.html) related to the impacts of human disturbance on biodiversity and my work is increasingly dependent on collaboration, data synthesis, and data science tools.

*How can I or with what I would like to contribute*:  I'm relatively new to GitHub (only started using in my postdoc 2 years ago) and excited to bring a beginner's mind to this paper. I may not have the most knowledge to share, but I enjoy writing and editing, and can help to craft an accessible, informative, concise paper.

*What is the most trepidation that I have*: Many cooks in the kitchen! But skilled ones, so I think we'll be okay, with Rob's great leadership.

**Reply by dylangomes** [reply]
Congratulations on the assistant professorship! That is HUGE!

**Comment by katherinehebert** [comment]
*My Name is*: Katherine Hébert

*What do I do*: I am a PhD candidate in the [Integrative Ecology Lab](https://ielab.recherche.usherbrooke.ca/) at Université de Sherbrooke in Québec, Canada. My PhD focuses on how how a multispecies perspective can help us measure biodiversity change more reliably - both in terms of indices of change, and our ability to interpret this change ecologically. I also coordinate an [R Workshop Series](https://r.qcbs.ca/) funded and hosted by the Québec Centre for Biodiversity Science with @pedrohbraga! :)

*How can I or with what I would like to contribute*: Writing, editing, and reviewing pull requests! 

*What is the most trepidation that I have*: Like most of you, a lack of time! I'm also excited to see a manuscript written collaboratively on GitHub though slightly nervous, which is a good sign because we're leaving our comfort zone here - what a neat experiment to be a part of! :)

**Comment by mburrus** [comment]
*My Name is*: Madison Burrus

*What do I do*: I am a Computer Systems Engineer at Lawrence Berkeley National Lab. Some of my responsibilities include managing project data management workflow, using best practices to store environmental data, using Application Programming Interfaces (APIs) to manage data pipelines (one-off work), and communicating these data management tools to others.

*How can I or with what I would like to contribute*: I have experiencing using GitHub and Zenboard to manage non-programming and programming tasks/issues (~2 years) and would like to use this experience to provide context to the paper where applicable. I'm most interested in understanding and communicating the overall workflow for scientific writing with GitHub, including figure/table design. 

*What is the most trepidation that I have*: Overcommitting to certain sections of the paper or underestimating the time it will take to research my chosen sections. Some of the investigations in the paper outline seem hard to pin down and I worry that I won't find (scientifically?) significant contributions. Also, merge conflicts. Frequent commits with this many people, keeping track of branches, timely reviews, and the subsequent merge conflicts....

**Comment by emmajhudgins** [comment]
_My Name is:_ Emma Hudgins

_What do I do:_ I'm a postdoc at Carleton University in Ottawa, Canada. I study invasive species, mostly those that attack trees, and increasingly those in urban contexts. I use simulation models and optimizations to predict their spread, impacts, and figure out best management practices.

_How can I or with what I would like to contribute:_ I'm passionate about building a scientific community that is less based on following the money/pressure and more based on openness and collaboration, so I'd like to bring that energy to the manuscript. I've done one perspectives paper before that I can hopefully draw from, and I've worked on a lot of big multiauthor collaborations through the InvaCost project (which is how i met @robcrystalornelas!)

_What is the most trepidation that I have:_ I'm hoping to use this experience to learn and solidify my relatively beginner-level github skills. Even going through all the documentation for this manuscript has been pretty daunting. I'm also still learning to juggle all the projects I've got going on with my postdoc, wrapping up PhD stuff I haven't published yet, and looking for next steps.

**Comment by dylangomes** [comment]
*My Name is*:  Dylan Gomes

*What do I do*:  I am currently a postdoctoral researcher at Oregon State University and NOAA Fisheries. While I am technically based out of Newport, Oregon, I have been working remotely for many months in Boulder, Colorado. I am building an end-to-end ecosystem model of the northern California Current marine ecosystem that will allow various climate change and management scenarios to be simulated. One goal of the project is to understand why so many juvenile Chinook salmon are dying (from endangered Columbia River populations). Previously I have spent a lot of time studying anthropogenic noise and artificial light at night and how it affects animal behavior, abundance, and activity. Most of my previous research has been with bats, birds, insects, and spiders.

*How can I or with what I would like to contribute*:  I mostly work solo in GitHub, and use it as a way to version control and store data and code, which I then make publicly available via Dryad or Zenodo. I am probably most useful in writing sections related to these things, but am also interested in general editing of the entire MS, and working on a table comparing various platforms researchers use for collaboration.

*What is the most trepidation that I have*: Currently it is lack of time for everything that I'd like to do. After working on my ecosystem modelling, meeting with collaborators, and looking for permanent positions, "side projects" like these tend to not get my immediate attention. Like Emma said in her post:

> I'm also still learning to juggle all the projects I've got going on with my postdoc

**Comment by adbinley** [comment]
My Name is: Allison Binley

What do I do: I am a PhD candidate at Carleton University specializing in quantitative biodiversity conservation. The aim of my thesis research is to examine how advances in the use and integration of big datasets are crucial for conducting more effective conservation research and improving management decisions and outcomes. My work focuses on integrating existing data, particularly community science data, to conduct broad-scale conservation research. I have a strong background in spatial statistics, decision science, data integration, literature review and big data analysis, and my research spans local, hemispheric and global scales.

How can I or with what I would like to contribute: Lately I have spearheaded a number of projects looking at maximizing the use of open data. I believe a lot of resources go to waste collecting more when we often have sufficient information to inform conservation action. Tools such as Github help facilitate the use of open data, which is what initially drew me to this project. I use Github to collaborate with other researchers, publish my code and data, build and host my website, and back up my workflow. I'm happy to contribute to any section that falls within my expertise.

What is the most trepidation that I have: I am very late to the party, so I hope that my contributions are helpful rather than tangential, given that I've missed most of the discussions. Although I have been using Github for a number of different purposes and am getting more familiar with it every day, I still frequently succeed in breaking it.

**Reply by wavingtowaves** [reply]
Really excited for your contributions to this project, Allison. Welcome!

**Comment by fhillemann** [comment]
*My Name is*: Friederike Hillemann [freddy; she/her]

*What do I do*: I'm a postdoctoral researcher at the Department of Human Behavior, Ecology and Culture, in Leipzig's Max Planck Institute for Evolutionary Anthropology. Currently, I work on projects that help understand Inuit subsistence ecology, from foraging decisions, to predictors of harvest success, and food sharing networks. For my PhD, I studied socio-ecological factors shaping mixed-species groups, using foraging flocks of wintering songbirds as a model system.

*How can I or with what I would like to contribute*: Contributing to sections that encourage others to use git (tips on how to get started), and to sections that touch on topics that are most relevant to my everyday work (mainly data storage and back up). Happy to be involved general editing.

*What is the most trepidation that I have*: Still being a newbie to git myself, the complexity of writing a collaborative manuscript using github can be daunting.

**Reply by wavingtowaves** [reply]
Yay Freddy! Welcome & excited for your collaboration on the manuscript--you will be a git expert in no time :)

**Comment by VLucet** [comment]
*My Name is*: Valentin (Val) Lucet

*What do I do*: I am an ecologist and research software engineer at Concordia University where I work software tools for ecosystem based management of fisheries.

*How can I or with what I would like to contribute*: I can contribute a section, and I am happy to re-read and review as well. For example, I have some experience teaching these tools would be happy to help describe what I perceive to be the biggest obstacles in teaching and how to help users gain confidence in the tool.

*What is the most trepidation that I have*: I'm coming in late on this project and I hope I can still contribute something to it!

**Reply by VLucet** [reply]
Also happy to review pull requests anytime.

**Comment by helenweierbach** [comment]
*My Name is*: Helen Weierbach

*What do I do*: I'm a Environmental Data Science research assistant working in EESA at Berkeley lab. Before the lab, I got my B.S. in mathematics, and now focus on developing statistical and physical models for predicting water quality and biogeochemistry. In addition to working with the EESA data group, I also work closely with the Ecology department. 

*How can I or with what I would like to contribute*: General comments/edits, as well as filling in any relevant holes I can help with related to using git/GitHub for managing ecological modelling codes, performing research, etc. 

*What is the most trepidation that I have*: Using GitHub in a new way, and figuring out how I can best 'suggest' edits and add comments editing in GitHub!
